src/style.css

```diff
@@ -1 +1,32 @@
 /* add styles here */
+.page__body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  width: fit-content;
+}
+
+.stars__star {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 0.5px;
```

Comment:
The `border-radius` value is very small (0.5px), which might not be noticeable. If this is intentional and you want to have a very slight rounding, that's fine. However, if you intended to have a more rounded effect, consider increasing the value.